user_interaction/static/themes/kinjin-theme.css

```diff
@@ -0,0 +1,77 @@
+@import url("https://fonts.googleapis.com/css2?family=Mochiy+Pop+One&display=swap");
+body {
+    font-family: "Mochiy Pop One", sans-serif;
+}
+/* Logo Knights verstoppen */
+.navbar-brand {
+    visibility: hidden;
+    width: auto;
+}
+/* add Kinjin Knights logo */
+.navbar-brand::before {
+    visibility: visible;
+    content: url(./images/kinjin/KinjinSpoofH70px.png);
```

Comment:
I would still like HDPI support for this.

Reply:
Moeite. Mag je ook zelf doen 😉